luxtronik/__init__.py

```diff
@@ -58,9 +119,15 @@ def write(self):
             LOGGER.debug("Command %s", cmd)
             val = struct.unpack(">i", self._socket.recv(4))[0]
             LOGGER.debug("Value %s", val)
-        self._disconnect()
-        # flush queue after writing all values
+        # Flush queue after writing all values
         self.parameters.queue = {}
+        # Give the heatpump a short time to handle the value changes/calculations:
+        time.sleep(WAIT_TIME_BETWEEN_WRITE_READ_PARAMETER)
+        # Read the new values based on our parameter changes:
+        self._read_parameters()
+        self._read_calculations()
+        LOGGER.warning("New: '%s'", self.calculations.get(12))
```

Comment:
I don't understand why reading and logging calculation 12 (`ID_WEB_Sollwert_TRL_HZ`) in a generic write method is needed.

Could this be a leftover from some testing?